primeNumber.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+var num=parseInt(process.argv[2],10);
+var status=0;
+    for(var n=2;n<num;n++){
```

Comment:
code having wrong indentation. 
please remote spaces from starting of line from line number 4 to 12.

Reply:
done